lib/steps.js

```diff
@@ -711,6 +711,10 @@ When(/I set the address of ?(?:the var(?:iable)?)? "([^"]+)" to "([^"]+)"/, asyn
 
 After(async function(scenario) {
   if (scenario.result.status !== 'passed') {
+    if ( env_vars.DEBUG ) {
+      console.log( 'Error occurred while running test' );
+      await scope.page.waitFor( 60 * 1000 );
```

Comment:
What's the reason behind waiting _after_ an error happens?

Reply:
Good question. It's so the developer (developing locally) has time to examine the page before it closes. I'll add that comment in the next PR.